### DISCUSSION TITLE
Lesson 14 GovToken.sol

### DISCUSSION THREAD

**Discussion by manu-acho** [Cyfrin/foundry-full-course-cu#1822] [Q&A]
I'm just wondering if anyone else noticed the code used in GovToken.sol (as copied from the openzeppelin wizard) as well as  ERC20Votes.sol is different (probably updated)  from the current version in the wizard and openzeppelin library (ERC20Votes.sol) ? For instance the _afterTokenTransfer() function is no longer provided. 
This makes following the rest of the lesson a little difficult and I am already quite apprehensive about testing.

**Comment by EngrPips** [comment]
Hello @manu-acho, There was an update. I ran into the same issue when I got to that section. You can try to figure it out; if not, let us know what you need help with. We are always here to help you, and also most of the updates that have happened since the course was released are always addressed in the `chronological-updates` tab of the Github repo associated with the course. Here `https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md` and see if the issue has been addressed already.

**Reply by manu-acho** [reply]
Thanks a lot,  I'd take a look.
Cheers!!

**Reply by EngrPips** [reply]
You are welcome. Looking forward to hearing the outcome.
